backends/platform/sdl/win32/win32_wrapper.cpp

```diff
@@ -23,12 +23,19 @@
 #define FORBIDDEN_SYMBOL_EXCEPTION_strcat
 
 #define WIN32_LEAN_AND_MEAN
-#include <windows.h>
-#include <shellapi.h> // for CommandLineToArgvW()
-#if defined(__GNUC__) && defined(__MINGW32__) && !defined(__MINGW64_VERSION_MAJOR)
```

Comment:
Why has this gcc check been removed?

Reply:
Previously, we assumed that MinGW32 set `_WIN32_IE` too low, and defined it high enough here. This PR changes it so that we check the actual value before redefining it, so we don't need the special case here any more.